plugin/dnstap/handler.go

```diff
@@ -18,12 +18,14 @@ type Dnstap struct {
 
 	// IncludeRawMessage will include the raw DNS message into the dnstap messages if true.
 	IncludeRawMessage bool
+	Identity []byte
+	Version []byte
 }
 
 // TapMessage sends the message m to the dnstap interface.
 func (h Dnstap) TapMessage(m *tap.Message) {
 	t := tap.Dnstap_MESSAGE
-	h.io.Dnstap(&tap.Dnstap{Type: &t, Message: m})
+	h.io.Dnstap(&tap.Dnstap{Type: &t, Message: m, Identity: h.Identity, Version: h.Version})
```

Comment:
Can you also extend handler test https://github.com/coredns/coredns/blob/0188090ae93557d02b7db25b13c83c86b412f172/plugin/dnstap/handler_test.go#L36 and verify identity and version is set as expected?